build/yaml/crd/vpc/crd.nsx.vmware.com_ipblocksinfoes.yaml

```diff
@@ -0,0 +1,83 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.14.0
+  name: ipblocksinfoes.crd.nsx.vmware.com
+spec:
+  group: crd.nsx.vmware.com
+  names:
+    kind: IPBlocksInfo
+    listKind: IPBlocksInfoList
+    plural: ipblocksinfoes
+    singular: ipblocksinfo
+  scope: Cluster
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: IPBlocksInfo is the Schema for the ipblocksinfo API
+        properties:
+          apiVersion:
+            description: |-
+              APIVersion defines the versioned schema of this representation of an object.
+              Servers should convert recognized schemas to the latest internal value, and
+              may reject unrecognized values.
+              More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
+            type: string
+          externalIPBlocks:
+            items:
+              properties:
+                cidr:
+                  description: A contiguous IP address space represented by network
+                    address and prefix length.
+                  type: string
+                visibility:
```

Comment:
Is `visibility` necessary if we're already under "externalIPBlocks"?

Reply:
Removed, thanks